ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/SingleEntityVulnerabilitiesTable.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Button, ButtonVariant } from '@patternfly/react-core';
+import { TableComposable, Thead, Tr, Th, Tbody, Td } from '@patternfly/react-table';
+import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons';
+import { SVGIconProps } from '@patternfly/react-icons/dist/js/createIcon';
+
+import LinkShim from 'Components/PatternFly/LinkShim';
+import SeverityIcons from 'Components/PatternFly/SeverityIcons';
+import { vulnerabilitySeverityLabels } from 'messages/common';
+import { getDistanceStrictAsPhrase } from 'utils/dateUtils';
+import { ImageVulnerabilitiesResponse } from './hooks/useImageVulnerabilities';
+import { getEntityPagePath } from './searchUtils';
+
+export type SingleEntityVulnerabilitiesTableProps = {
```

Comment:
This table is almost identical in both the image and deployment single pages, with the exception that the image name is displayed as a column next to image components in the deployment page. The way the gql resolvers are structured make it so that we will need to do some pre-processing of the data on that page before feeding it into this table.

e.g. The query might looks something like
```graphql
images {
    imageVulnerabilities {
        imageComponents {}
    }
}
```
so we would need to get the image metadata from the top level and pass it down to the image components to get the UI structure we need.

Mandar is looking into possible alternate queries for this, so I didn't generalize the prop type here too much in order to avoid having to change it a lot later.